dist/js/app.js

```diff
@@ -1,152 +1,372 @@
+            .html(null)
+            .append("div")
+            .classed("row",true);
+
+        housing.endpoints.loadAdmin().then(
+            function(data){
+                data = data.sort(housing.admin.style.listSorter);
+                housing.admin.init(main,nav,data);
+                housing.admin.load(data);
+            });
+    } else {
+        // If not signed in, clear navigation and insert signin button.
+        nav.html(null);
+        nav.append("a")
+            .classed("button",true)
+            .text("Sign In")
+            .on("click",housing.endpoints.click);
+        // message for IE users
+        nav.append("p").text("Note: there is a current bug with Google Sign In and Internet Explorer.  If you are using Internet Explorer and you end up with a blank window after sigining in, close the window and click Sign In again.");
+        $("#loading").hide();
+    }
+}
+    
 /**
- * Loads Google APIs, attempts to log in the user and calls the provided
- * callback function with the results of the authentication attempt.
+ * housing.admin.init
+ *
+ * Creates navigation buttons 
+ *
+ * @param main       An HTML element (with class row) in which to place lists
+ * @param nav        An HTML element where navigation buttons are placed
+ * @param data       A JSON element containing the rooms and floors
  */
+housing.admin.init = function(main,nav,data) {
+    console.log("Initializing Housing Admin")
+    housing.admin.mainelement = main;
+    // Create navigation
+    if( nav ) {
+        // Clear old stuff
+        nav.html(null);
+        
+        // Load open/closed state of form
+        housing.endpoints.isOpen().then(function(resp){
+            // Don't add any components until we know that this is supported
+            var switch_container = nav.append("div")
+                .classed("row",true);
+
+            var switch_fieldset = switch_container.append("div")
+                .classed("small-2 columns",true)
+            .append("fieldset")
+                .classed("switch round large",true)
+                .attr("tabindex",0);
+            var switch_checkbox = switch_fieldset.append("input")
+                .attr("type", "checkbox")
+                .attr("id", "openswitch")
+                .attr("checked", "checked");
+            switch_fieldset.append("label")
+                .attr("for","openswitch");
+
+            housing.admin.isOpen = resp.result.isOpen
+            var labeltext = housing.admin.isOpen ? "Open" : "Closed";
+            switch_container.append("div")
+                .classed("switch-label small-10 columns", true)
+                .attr("id", "openswitch-label")
+                .html("Housing Form "+labeltext);
+        },function(resp){
+            switch(resp.result.error.code){
+                case 401: 
+                    housing.endpoints.errorHelper(resp.result.error,'isOpen()'); break;
+                case 404: case 600: 
+                    break;
```

Comment:
This is to make it so that it ignores the fact that the `isOpen` method doesn't exist yet.